<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWidget</class>
 <widget class="QWidget" name="InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLineEdit" name="inputEdit">
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Input file...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inputBrowseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>404</width>
        <height>415</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="mediaInfosText">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="customParamsLayout">
         <property name="horizontalSpacing">
          <number>3</number>
         </property>
         <property name="verticalSpacing">
          <number>3</number>
         </property>
         <item row="0" column="1">
          <widget class="QComboBox" name="frameRateBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>5</number>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Custom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23.976 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>29.97 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>59.94 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>60 fps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>120 fps</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="frameRateEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> fps</string>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>24.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="addParamButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Custom option...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="frameRateButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Frame rate</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="trcButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Color transfer (Gamma)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="trcBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
